doc/api/http2.md

```diff
@@ -984,14 +989,16 @@ added: v8.4.0
 The `'timeout'` event is emitted after no activity is received for this
 `Http2Stream` within the number of milliseconds set using
 `http2stream.setTimeout()`.
+Its listener does not expect any arguments.
 
 #### Event: 'trailers'
 <!-- YAML
 added: v8.4.0
 -->
 
 The `'trailers'` event is emitted when a block of headers associated with
-trailing header fields is received. The listener callback is passed the
+trailing header fields is received. Its listener does not expect any arguments.
```

Comment:
The next sentence says it is passed an argument, does it not?

Reply:
yes, listener callback. okay i add them thanks @Trott